packages/migration/src/migrations/hearth/20230602120944-populate-valid-locationid.ts

```diff
@@ -150,10 +151,10 @@ async function updateEventLocationIdOrJurisdictionIds(db: Db, elasticDoc: any) {
     )
 
     if (locationDoc.length > 0) {
-      if (locationDoc[0].type.coding[0].code === 'HEALTH_FACILITY') {
-        body.eventLocationId = locationDoc[0].id
+      if (locationDoc?.[0]?.type?.coding?.[0]?.code === 'HEALTH_FACILITY') {
```

Comment:
```suggestion
      if (locationDoc[0].type?.coding?.[0]?.code === 'HEALTH_FACILITY') {
```
as we just asserted that length is greater than zero :smile: